__init__.py

```diff
@@ -139,7 +139,7 @@ def handle_take_pic_intent(self, message):
             LOG.info("In picture")
             LOG.debug(message.data)
             today = datetime.datetime.today()
-            user = get_message_user(message)
+            user = get_message_user(message) or os.environ.get('USER', os.environ.get('USERNAME'))
```

Comment:
This is looking for `username` in a different scope, but I like this pattern to default to envvars since that is more specific than a hard-coded default and should be consistent across skills/services (other places I use "local"). I might borrow this in some other places :)

Reply:
I realize it is not necessarily looking for the os user, but I thought it was a good default when none is sent